shader/binning.wgsl

```diff
@@ -89,12 +89,18 @@ fn main(
         let bbox_raw = bbox_intersect(clip_bbox, pb);
         // TODO(naga): clunky expression a workaround for broken lhs swizzle
         let bbox = vec4(bbox_raw.xy, max(bbox_raw.xy, bbox_raw.zw));
```

Comment:
You probably no longer need the max here, as everything downstream should be able to interpret x1 < x0 or y1 < y0 as an empty box.

Reply:
Done.